src/components/shared/seo/canonical-url/canonical-url.jsx

```diff
@@ -0,0 +1,17 @@
+'use client';
```

Comment:
The same question about client code. I guess it's `usePathname`. Are you sure we can't replace it for something what works in build time?

Reply:
Yes, we can. I'll update it in the next iteration.